spyder/config/main.py

```diff
@@ -158,6 +158,7 @@
               'startup/use_run_file': False,
               'startup/run_file': '',
               'greedy_completer': False,
+              'ipy_jedi_completer': False,

```

Comment:
Please change this to `jedi_completer`. We already know that it's an IPython console option because it's in the right section.